### ISSUE TITLE
Content Downloading does not work

### ISSUE THREAD

**Issue by Matias-Barrios** [OpenTTD/OpenTTD#8582]
## Version of OpenTTD
Open TTD 1.10.3

## Expected result
Being able to download mods, music, stuff from "Check online content" feature

## Actual result
The list never loads. Its always empty.

## Steps to reproduce
Open the game and go to "Check online content"

This is all I see when I go there :

![image](https://user-images.githubusercontent.com/30472800/104831377-1f9a1700-5867-11eb-8ca8-698446bd8d75.png)

The list never loads. No matter what.

**Comment by James103** [comment]
Are you behind a proxy/VPN? If so, can you try connecting directly without the proxy / VPN?

**Comment by Matias-Barrios** [comment]
Hi @James103 . Thanks for replying. Im not in a VPN or anything like that. This issue occurs in both the Ubuntu 20.04 I got installed as well as in the Android version I have. 

I am in Latam. Uruguay specifically and I am not using any vpn.

**Comment by TrueBrain** [comment]
Hi @Matias-Barrios ,

I just checked, it is sadly not a problem on our end. Why sadly? Because it means it is most likely something on your end, and that is really difficult to debug from here. But I am not going to leave it at that, so let me name a few of the common issues and let's see if we can find the issue for you :)

Our infrastructure is hosted on AWS. So first we need to make sure you can reach AWS. Can you visit the following websites:
- https://www.openttd.org/ <- shows our main frontpage
- https://content.openttd.org/ <- shows a completely blank page

If both work, it means you can reach AWS, and the most common cause next is that the custom port we use for the Content Service is blocked. Can you check if you have a firewall enabled and if so, if it is allowing through port `3978` on TCP, outbound?

Next up, can you check if you can reach the Master Server, by clicking Multiplayer, making sure "Advertised" is on, and clicking Find Server. Does this show servers?
If it does show servers, can you join any server and play for a bit? If only for 1 minute, but the fact that you can connect and play tells a lot :)

If that all works out, the next case would be IPv6. Do you use IPv6? A simple website to find out is: https://ipv6-test.com/ . It will tell you what your IPv4 and IPv6 is. If you have IPv6, can you disable that and try again? Recently we fixed some issues for people with both IPv6 and IPv4, where one of the two is not working as intended (for various of reasons, IPv6 is not always as stable as you might think). Can you try a recent nightly and see if that fixes the problem? You can find our nightlies here: https://www.openttd.org/downloads/openttd-nightlies/latest.html

And as final hail marry, there are known cases where your ISP is filtering ports. This happens in some countries and for some ISPs, where they only allow certain ports to communicate to the internet. They can block random ports. If this is the case, that is really hard to test and find out ... but I just mention it as we have seen this in the past.

In short, if you can test all the above, that would greatly help us narrow down where potentially an issue is. To give a simple checklist to follow:

- [ ] https://www.openttd.org/ shows OpenTTD's frontpage
- [ ] https://content.openttd.org/ shows a full blank page (no error)
- [ ] Firewall allows `3978` outbound
- [ ] Multiplayer -> Find Server works
- [ ] Playing on a multiplayer server works
- [ ] I have IPv6 and IPv4
- [ ] Disabling IPv6 did not help
- [ ] Nightly is also not working

A lot, I know .. but as mentioned, as it is working for us, and there are no known issues .. sadly we have to ask you to do all these tests :( Sorry about that.

**Comment by Matias-Barrios** [comment]
Thanks so much @TrueBrain and @James103 ! I disabled IPV6 and now everything works as charm!
You can close this issue. Totally fixed!! :100: :+1:
